src/client/react/components/CustomFunctionsDashboard/Summary/Summary.tsx

```diff
@@ -1,21 +1,33 @@
 import * as React from 'react';
 import styled from 'styled-components';
-
 import PivotContentContainer from '../PivotContentContainer';
 import List, { Item } from '../List';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
 
 const TopInfo = styled.div`
   padding: 27px 24px 0px 17px;
 `;
 
-const FunctionsContainer = styled.div`
+const ErrorContainer = styled.div`
   height: 100%;
   overflow-x: hidden;
   overflow-y: auto;
   flex-shrink: 2;
   border-top: 1px solid #f4f4f4;
 `;
 
+const RegisteredFunctionsContainer = styled.div`
+
+  width: 100%
+  height: auto;
+  overflow-x: hidden;
+  overflow-y: auto;
+  flex-shrink: 2;
+  border-top: 1px solid #f4f4f4;
+  z-index: 2000;
+  bottom: 20px;
+`;
+
 const Summary = ({ metadata }: { metadata: ICFVisualMetadata }) => {
```

Comment:
Component is becoming way too large, break into smaller components (see image I messaged you)